cli/.idea/misc.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think editor setup shouldn't be commited to the repo. Would you mind adding this to the `.gitignore`?